public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -505,14 +512,22 @@ export const AnomalyDetailsChart = React.memo(
                       const end = get(
                         brushArea,
                         'x.1',
-                        DEFAULT_DATE_PICKER_RANGE.start
+                        DEFAULT_DATE_PICKER_RANGE.end
                       );
                       handleZoomRangeChange(start, end);
                       if (props.onDatePickerRangeChange) {
                         props.onDatePickerRangeChange(start, end);
                       }
                     }}
                     theme={ANOMALY_CHART_THEME}
+                    xDomain={
+                      showAggregateResults
+                        ? undefined
```

Comment:
an error will be thrown if we set an `xDomain` in the aggregate anomalies case, since the scale type (see lines 633-635) won't work properly in the `ScaleType.Ordinal` type